### DISCUSSION TITLE
How can we set a value of a field1 with the value of field2 in the same form

### DISCUSSION THREAD

**Discussion by venkateshpullaganti** [manojadams/mui-forms#65] [Q&A] [answered]
Hi @manojadams,
I tried searching in the docs but couldn't find it. In `prop_update` is there a way so that we can fill the field value based on the other field's?

**Comment by manojadams** [comment]
Yes, this can be done with `propUpdate` but currently it is conditional.

I am currently working on an update to standardise the schema regarding this.

Will update once it is available.

**Reply by venkateshpullaganti** [reply]
Okey Sure @manojadams 

Let me know I you need a hand.

**Reply by manojadams** [reply, marked as answer]
Here:
https://mui-forms.vercel.app/field-dependencies/prop-update
